egs/mgb2_arabic/s5/local/train

```diff
@@ -0,0 +1,2214 @@
+train/62C90151-AC4B-4D11-A9A2-B70DC88EED8A
+train/259DD50E-E65B-46A6-A419-9FC90F087D4E
```

Comment:
There isn't a more compact way to specify t he training split?  I'm just concerned about adding bulk to the git repo.